scripts/get_nightly_version.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+#------------------------------------------------------------------------------
+# Prints the exact version string that would be used to describe a nightly
+# build of the compiler.
+#
+# The documentation for solidity is hosted at:
+#
+#     https://solidity.readthedocs.org
```

Comment:
I guess that's here because it's also in get_version.sh ? I'm not entirely sure why it's in there and why it should be in here - feels like a rather random piece of information to me...

Reply:
Seemed out of place to me too but I've seen it used in several other files (e.g. [build_emscripten.sh](https://github.com/ethereum/solidity/blob/v0.6.10/scripts/build_emscripten.sh) or [docs.sh](https://github.com/ethereum/solidity/blob/v0.6.10/scripts/docs.sh)) so I thought it's a deliberate convention. Should I remove it?

Docstrings and license comments are a mess in scripts anyway. Some have them like this, some do it differently, some do not have them at all.

Reply:
Yeah.. I'd say we can remove it here, since it seems rather weird - and maybe we should bring it up quickly in a meeting and decide on a convention for all the scripts and unify them in that regard... not that any of this is all that important, but still :-).